languagetool-language-modules/es/src/main/resources/org/languagetool/rules/es/grammar.xml

```diff
@@ -12569,6 +12569,18 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         </rulegroup>
     </category>
     <category id="MISSPELLING" name="Ortografía" type="misspelling">
+        <rule id="EX_ALTO_CARGO" name="exalto cargo (ex alto cargo)" default="temp_off">
+            <pattern>
+                <marker>
+                    <token>exalto</token></marker>
+                <token regexp="yes">cargo|mando</token>
+            </pattern>
+            <message>La expresión <suggestion>ex alto</suggestion> <match no="2"/> se escribe en tres palabras.</message>
+            <url>https://www.fundeu.es/recomendacion/ex-alto-cargo-en-tres-palabra/</url>
```

Comment:
This message will look strange: `La expresión  «ex alto» cargo se escribe en tres palabras`.
Do this: 

```
<message>La expresión "ex alto <match no="2"/>" se escribe en tres palabras.</message>
<suggestion>ex alto</suggestion>

```